addons/stock/tests/test_stock_return_picking.py

```diff
@@ -36,7 +36,7 @@ def test_stock_return_picking_line_creation(self):
             'location_id': self.stock_location,
             'picking_id': picking_out.id,
         })
-        return_wizard._onchange_picking_id()
+        return_wizard._compute_moves_locations()
```

Comment:
You shouldn't need to call computes directly. The onchanges were called directly because they aren't auto-called unless used in a form view (i.e. you can delete this line)